core/src/main/scala/cats/data/Kleisli.scala

```diff
@@ -131,6 +131,19 @@ private[data] sealed trait KleisliFunctions {
   def liftK[F[_], A]: F ~> Kleisli[F, A, ?] =
     λ[F ~> Kleisli[F, A, ?]](Kleisli.liftF(_))
 
+  /**
+    * Same as [[apply]], but expressed as a FunctionK for use with mapK
+    * {{{
+    * scala> import cats._, data._, implicits._
+    * scala> def f(a: Int): Option[Char] = if (a > 0) Some('y') else None
+    * scala> val b: Kleisli[Option, Int, Char] = Kleisli[Option, Int, Char](f _)
+    * scala> b.mapK( Kleisli.applyK(1) )
```

Comment:
The `mapK` on `Kleisli` takes an `F ~> G` to transform an `Kleisli[F, A, B]` into `Kleisli[G, A, B]`.

It seems you are trying to use the `mapK` originating from cats-tagless `FunctorK`.